CHANGELOG.md

```diff
@@ -2,34 +2,58 @@
 
 ## Current Develop Branch
 
+## 9.0.0 Fri Jan 8 2021
+
+- [#9156](https://github.com/MetaMask/metamask-extension/pull/9156): Remove window.web3 injection
+- [#10039](https://github.com/MetaMask/metamask-extension/pull/10039): Add web3 shim usage notification
+- [#8640](https://github.com/MetaMask/metamask-extension/pull/8640): Implement breaking window.ethereum API changes
+- [#8629](https://github.com/MetaMask/metamask-extension/pull/8629): Fix `eth_chainId` return values for Infura networks
+- [#10019](https://github.com/MetaMask/metamask-extension/pull/10019): Increase Chrome minimum version to v63
+- [#10135](https://github.com/MetaMask/metamask-extension/pull/10135): Fix error where a swap only completed the token approval transaction
+- [#10100](https://github.com/MetaMask/metamask-extension/pull/10100): Remove unnecessary swaps footer space when in dropdown mode
+- [#9905](https://github.com/MetaMask/metamask-extension/pull/9905): Redesign view quote screens
+- [#9320](https://github.com/MetaMask/metamask-extension/pull/9320): Prevent hidden tokens from reappearing
+- [#10000](https://github.com/MetaMask/metamask-extension/pull/10000): Use consistent font size for modal top right Close links
+- [#10046](https://github.com/MetaMask/metamask-extension/pull/10046): Improve home screen notification appearance
+- [#10117](https://github.com/MetaMask/metamask-extension/pull/10117): Fix network settings Kovan block explorer link
+- [#10143](https://github.com/MetaMask/metamask-extension/pull/10143): Prevent malformed next nonce warning
+- [#10142](https://github.com/MetaMask/metamask-extension/pull/10142): Update @metamask/contract-metadata from v1.20.0 to 1.21.0
```

Comment:
Nit: we should update this to include https://github.com/MetaMask/metamask-extension/pull/10093, which has the effect of allowing rollback to a custom RPC endpoint when failing to connect to a network.

Reply:
Done in: 5ebc6bf